core/string_api.php

```diff
@@ -666,7 +667,15 @@ function string_get_bug_view_link( $p_bug_id, $p_detail_info = true, $p_fqdn = f
 				$t_link .= ' class="resolved"';
 			}
 		}
-		$t_link .= '>' . bug_format_id( $p_bug_id ) . '</a>';
+		$t_link .= '>' . bug_format_id( $p_bug_id );
+		if ( $p_long_link ) {
+			$t_short_summary = $t_summary;
+			$t_summary_length = config_get( 'timeline_issue_summary_length' );
+			$t_short_summary = mb_strimwidth( $t_summary, 0, $t_summary_length, '...', 'utf-8' );
```

Comment:
Remove 'utf-8' as we have set mb_internal_encoding.

BTW @dregad, I am note complete sure at the moment if we still have to check for `extension_loaded('mbstring')` or if we assume that the extension is always enabled.
At least during installation you made it mandatory since 1.3.5, see https://www.mantisbt.org/bugs/view.php?id=21959 but upgraded old installations might still run without mbstring enabled.